src/app/proposals/components/table-columns/CategoryColumn.tsx

```diff
@@ -1,24 +1,25 @@
 import { Tooltip } from '@/components/Tooltip'
-import { GrantsIcon, ToolsIcon, Milestone1Icon, Milestone2Icon, Milestone3Icon } from './icons'
+import { GrantsIcon, HammerIcon, Milestone1Icon, Milestone2Icon, Milestone3Icon } from '../icons'
 import { ProposalCategory } from '@/shared/types'
 import { ClassNameValue } from 'tailwind-merge'
 import { cn } from '@/lib/utils'
 
-const icons = {
-  [ProposalCategory.Grants]: <GrantsIcon />,
-  [ProposalCategory.Activation]: <ToolsIcon />,
-  [ProposalCategory.Deactivation]: <ToolsIcon />,
-  [ProposalCategory.Milestone1]: <Milestone1Icon />,
-  [ProposalCategory.Milestone2]: <Milestone2Icon />,
-  [ProposalCategory.Milestone3]: <Milestone3Icon />,
-} as const satisfies Record<ProposalCategory, React.JSX.Element>
-
 interface Props {
   category: ProposalCategory
   className?: ClassNameValue
+  hasGradient?: boolean
 }
 
-export function CategoryColumn({ category, className }: Props) {
+export function CategoryColumn({ category, className, hasGradient = false }: Props) {
+  const icons = {
+    [ProposalCategory.Grants]: <GrantsIcon hasGradient={hasGradient} />,
+    [ProposalCategory.Activation]: <HammerIcon hasGradient={hasGradient} />,
+    [ProposalCategory.Deactivation]: <HammerIcon hasGradient={hasGradient} />,
+    [ProposalCategory.Milestone1]: <Milestone1Icon hasGradient={hasGradient} />,
+    [ProposalCategory.Milestone2]: <Milestone2Icon hasGradient={hasGradient} />,
+    [ProposalCategory.Milestone3]: <Milestone3Icon hasGradient={hasGradient} />,
```

Comment:
instead of having 3 different icons, you could take component that I made. It accepts a digit parameter and renders icon dynamically .
https://github.com/RootstockCollective/dao-frontend/blob/main/src/app/proposals/components/MilestoneIcon.tsx
So, in this case you could do something like
[ProposalCategory.Milestone1]: <MilestoneIcon digit={'1'} hasGradient={hasGradient} />,
...